File: src/app/Entity/BPUser.ts
export class BPUser {

    bpuId:number;	
	bpufName:string;
	bpulName:string;
    bpuCompanyName:string;	
	bpuUsername:string;	
	bpuPassword:string;	
	bpuEmail:string;	
	bpuContactno:number;	
	bpuDOB:Date;

    constructor(bpuId:number, bpufName:string, bpulName:string, bpuCompanyName:string, bpuUsername:string,	bpuPassword:string, 
        bpuEmail:string, bpuContactno:number, bpuDOB:Date) {
        
            this.bpuId=bpuId;	
            this.bpufName=bpufName;
            this.bpulName=bpulName;
            this.bpuCompanyName=bpuCompanyName;
            this.bpuUsername=bpuUsername;	
            this.bpuPassword=bpuPassword;	
            this.bpuEmail=bpuEmail;	
            this.bpuContactno=bpuContactno;	
            this.bpuDOB=bpuDOB;

    }
}

File: Backend/src/main/java/com/boot/entity/BPUser.java
package com.boot.entity;

import java.sql.Date;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class BPUser {
	@Id
	@GeneratedValue
	@Column(name = "bpuId")	
	private int bpuId;
	
	@Column(name = "bpufName")	
	private String bpufName;
	
	@Column(name = "bpulName")	
	private String bpulName;
	
	@Column(name = "bpuCompanyName")
	private String bpuCompanyName;
	
	@Column(name = "bpuUsername")	
	private String bpuUsername;
	
	@Column(name = "bpuPassword")	
	private String bpuPassword;
	
	@Column(name = "bpuEmail")	
	private String bpuEmail;
	
	@Column(name = "bpuContactno")	
	private double bpuContactno;
	
	@Column(name = "bpuDOB")	
	private Date bpuDOB;
	
}
